superset-frontend/packages/superset-ui-core/src/chart/models/ChartProps.ts

```diff
@@ -76,10 +76,7 @@ export interface ChartPropsConfig {
   annotationData?: AnnotationData;
   /** Datasource metadata */
   datasource?: SnakeCaseDatasource;
-  /**
-   * Formerly called "filters", which was misleading because it is actually
-   * initial values of the filter_box and table vis
-   */
+  /** Initial values */
```

Comment:
Nit: maybe we should just remove this comment?

Reply:
Removed!